.github/workflows/test.yml

```diff
@@ -45,34 +45,52 @@ jobs:
             (echo; echo "Unexpected difference in directories after code generation. Run 'go generate ./...' command and commit."; exit 1)
 
   # Run acceptance tests in a matrix with Terraform CLI versions
-  # TODO: Re-enable acceptance tests: set TF_ACC to 1.
   test:
     name: Terraform Provider Acceptance Tests
     needs: build
     runs-on: ubuntu-latest
     timeout-minutes: 15
+
+    services:
+      tharsis:
+        image: registry.gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-all-in-one-docker
+        ports:
+          - 8000:8000 # API
+          - 8080:8080 # Keycloak
+        options: >-
+          --health-cmd "curl -f http://localhost:8000/health"
+          --health-interval 30s
+          --health-timeout 10s
+          --health-retries 5
+          --health-start-period 30s
+
     strategy:
       fail-fast: false
       matrix:
         # list whatever Terraform versions here you would like to support
         terraform:
-          - '1.0.*'
-          - '1.1.*'
-          - '1.2.*'
-          - '1.3.*'
+          - "1.0.*"
+          - "1.1.*"
+          - "1.2.*"
+          - "1.3.*"
```

Comment:
We might as well add the newer cli versions to this list while it's being updated.

Reply:
The latest CLI version is [1.37](https://github.com/hashicorp/terraform/releases). Won't the wildcard at the end cover it?

Reply:
That's my mistake, I misread that version.